### PR TITLE
fix(rangeSlider): error when no result

### PR DESCRIPTION
**Pull request by maxiloc**

### CODE REVIEW

**Comment by vvo** [comment]
Need test :)

**Comment by maxiloc** [comment]
done

File: widgets/range-slider/__tests__/range-slider-test.js [hunk 1]
```diff
@@ -80,86 +47,157 @@ describe('rangeSlider()', () => {
...
+        templateProps: {
+          templates: {footer: '', header: ''},
+          templatesConfig: undefined,
+          transformData: undefined,
+          useCustomCompileOptions: {footer: false, header: false}
+        },
+        tooltips: true
+      };
+
+      expect(ReactDOM.render.calledOnce).toBe(true, 'ReactDOM.render called twice');
```
Comment (bobylito) [inline widgets/range-slider/__tests__/range-slider-test.js]: Message should reflect the fact that we don't know the number of times it was called  : ReactDOM.render called 0 or more than 1 times :)

File: widgets/range-slider/__tests__/range-slider-test.js [hunk 2]
```diff
@@ -80,86 +47,157 @@ describe('rangeSlider()', () => {
...
+      expect(autoHideContainer.calledOnce).toBe(true, 'autoHideContainer called once');
```
Comment (bobylito) [inline widgets/range-slider/__tests__/range-slider-test.js]: Isn't the purpose of the message is to indicate the error we detected? In this case, wouldn't it be like on the previous line?

**Comment by maxiloc** [comment]
@bobylito Fix you first remark. Did not understand the second one

**Comment by bobylito** [comment]
Nevermind, I think it was wrong :D 

LGTM!

**Pull request merged**